_posts/2020-05-26-exploring-the-open-sgid-a-brief-tutorial-using-open-source-python-tools.md

```diff
@@ -15,27 +15,28 @@ tags:
+
+The Open SGID also creates an opportunity for users to pull in data and perform analysis using only open source Python libraries. With this route, you don't need to worry about paying for licensed software or even installing GIS desktop software at all. And if you use a cloud-hosted Jupyter notebook—like Google Colab—you don't even have to install Python on your computer (or tablet or chromebook)!
 
 ### Python Notebook and Libraries
 
-The rest of this blog will walk you through a few examples of how easy it can be to pull SGID data into Python to perform analysis or create simple maps.  These basic building blocks, combined with a little gumption and googling, can put you well on your way to performing whatever analysis you desire.  A link to the Jupyter notebook in Google Colab is also included, so you can run the code yourself and build upon it.  The following Python libraries will be used in this tutorial:
+The rest of this article will walk you through a brief tutorial of how easy it can be to pull SGID data into Python to perform analysis or create simple maps. These basic building blocks, combined with a little gumption and googling, can put you well on your way to performing whatever analysis you desire. A link to the Jupyter notebook in Google Colab is also included, so you can run the code yourself and build upon it.
 
--   [Psycopg2](https://www.psycopg.org/) - A PostgreSQL/PostGIS client for Python
--   [GeoPandas](https://geopandas.org/) - An open source Python project for working with geospatial data
-  - Extends the popular [Pandas](https://pandas.pydata.org/) library to geospatial data
-- [Matplotlib](https://matplotlib.org/) - a plotting and visualization library for Python
+*  [Psycopg2](https://www.psycopg.org/) - A PostgreSQL/PostGIS client for Python
+*  [GeoPandas](https://geopandas.org/) - An open source Python project for working with geospatial data
+  * Extends the popular [Pandas](https://pandas.pydata.org/) library to geospatial data
+* [Matplotlib](https://matplotlib.org/) - A plotting and visualization library for Python
```

Comment:
**chore**: please use a hyphen with one space between it and the text

Reply:
done